src/components/OIO/index.js

```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react'
 import classNames from 'classnames'
+import defaults from './defaults'
 import style from './style.less'
 
+// TODO: Deprecate fontFamily and primaryColor props
```

Comment:
@stevenyuen Any reason not to deprecate `primaryColor` and `fontFamily` in this PR? (At least internally)